bbc/simorgh
# WSTEAM1-660 - Remove Next.js test JSON output

Resolves JIRA [WSTEAM1-660](https://jira.dev.bbc.co.uk/browse/WSTEAM1-660)

Overall changes
======
- Removes rendered JSON output from the Next.js Live pages, as this was used for debugging purposes and is no longer needed

Code changes
======
- Removed rendered JSON elements
- Removed props and types only used for debugging on the page
- Removed styles for styling the JSON output
- Removed logger that logged a Next.js client render that was only added to check rendering was happening correctly in the Next.js app

Testing
======
1. Visit a Live page, e.g: https://www.test.bbc.com/pidgin/live/c7p765ynk9qt
2. Confirm that the test JSON output is now removed from the bottom of the page

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
